vcd/resource_vcd_rde_type_test.go

```diff
@@ -0,0 +1,241 @@
+//go:build rde || ALL || functional
+
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"regexp"
+	"strings"
+	"testing"
+)
+
+func TestAccVcdRdeType(t *testing.T) {
+	preTestChecks(t)
+	skipIfNotSysAdmin(t)
+
+	var params = StringMap{
+		"ProviderVcdSystem":   providerVcdSystem,
+		"ProviderVcdOrg1":     providerVcdOrg1,
+		"Nss":                 "nss",
+		"Version":             "1.0.0",
+		"Vendor":              "vendor",
+		"Name":                t.Name(),
+		"Description":         "Created by " + t.Name(),
+		"InterfaceReferences": "vcd_rde_interface.rde_interface1.id",
+		"SchemaPath":          getCurrentDir() + "/../test-resources/rde_type.json",
+		"SchemaUrl":           "https://raw.githubusercontent.com/adambarreiro/terraform-provider-vcd/add-rde-support-2/test-resources/rde_type.json", // FIXME
```

Comment:
Reminder to set final URL here. Or if you have any other suggestion, let me know

Reply:
We need your bot to capture "https://raw.githubusercontent.com" links :)

Reply:
Changed, time to merge.